.vscode/settings.json

```diff
@@ -14,5 +14,6 @@
   "eslint.enable": true,
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
-  }
+  },
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
What's this for?

Reply:
This probably needs reverting. My VSCode kicked up a fuss and wouldn't use the correct typescript version without this. It basically tells VSCode to use the project typescript version instead of the system version. I can remove it, or if it seems useful it can stay?

Reply:
I made some changes to the ts version recently which is probably related to this. Shall we move this change into a separate PR though so we can figure out if we need it set at this level or not? I'm not really sure at the moment to be honest and I don't think we should hold up this Source change for it